cfme/tests/cloud_infra_common/test_retirement.py

```diff
@@ -52,15 +58,39 @@ def test_vm(small_template, provider):
         logger.warning('Failed to delete vm from provider: {}'.format(vm.name))
 
 
+@pytest.yield_fixture(scope="function")
+def test_ec2_s3_vm(provider):
```

Comment:
because the other one is also named test_vm   -  I'm going to change this, good call